india_compliance/gst_india/client_scripts/purchase_receipt.js

```diff
@@ -0,0 +1,21 @@
+const DOCTYPE = "Purchase Receipt";
+setup_e_waybill_actions(DOCTYPE);
+
+frappe.ui.form.on(DOCTYPE, {
+    refresh(frm) {
+        if (!gst_settings.enable_e_waybill || !gst_settings.enable_e_waybill_from_pr)
+            return;
+        show_sandbox_mode_indicator();
+    },
+
+    after_save(frm) {
+        if (is_e_waybill_applicable(frm) && !is_e_waybill_generatable(frm))
```

Comment:
check for address also because in generatable we are also checking for same gstin.